File: backend/src/main/java/ru/home/shop/utils/UuidUtils.java
package ru.home.shop.utils;

import com.fasterxml.uuid.Generators;
import com.fasterxml.uuid.impl.TimeBasedGenerator;

import java.util.UUID;

public class UuidUtils {

    private static TimeBasedGenerator generator = Generators.timeBasedGenerator();

    public static UUID newUUID() {
        return generator.generate();
    }
}

File: backend/src/main/kotlin/io/github/artemy/osipov/shop/utils/UuidUtils.kt
package io.github.artemy.osipov.shop.utils

import com.github.f4b6a3.ulid.UlidCreator
import java.util.UUID

object UuidUtils {

    fun newUUID(): UUID {
        return UlidCreator.getUlid().toUuid()
    }
}
